StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/PaneLayoutView.swift

```diff
@@ -21,8 +21,24 @@ final class PaneLayoutView {
     private let paneLayoutView: UIView
     let scrollView: UIScrollView
 
-    init(contentView: UIView, footerView: UIView?) {
+    private var footerView: UIView?
+    private var safeAreaLayoutGuide: UILayoutGuide?
+    private var footerViewBottomConstraint: NSLayoutConstraint?
+
+    static var shouldMoveFooterViewAboveKeyboard: Bool {
+        // Only move the footer view on iPhones.
+        // The entire sheet is already shifted up with the keyboard on iPads.
+        UIDevice.current.userInterfaceIdiom != .pad
```

Comment:
does this act OK if the app is running in a slide out, split screen, or with a small window on an M-series iPad using stage manager?